src/gui/mainwindow.cpp

```diff
@@ -326,6 +326,15 @@ MainWindow::MainWindow(QWidget *parent)
     m_ui->actionCheckForUpdates->setVisible(false);
 #endif
 
+#ifdef Q_OS_MAC
```

Comment:
Does not change anything in Plasma, Qt documentation is unambiguous. I have no experience of Qt changing a documented feature, even between major releases.